### DISCUSSION TITLE
Manage Action to open href Link

### DISCUSSION THREAD

**Discussion by emilioaray-dev** [sandstreamdev/react-swipeable-list#210] [General]
Hi  Is it possible that the action when swipe is to open an external link?

`
<SwipeableList>

     <SwipeableListItem
           swipeLeft={{
               content:
                   <img src="/pages/any.svg" />,
               action: ( ) => #open link href="https://anyadress.com/" target="_blank#  }}
               onSwipeProgress={progress => console.info(`Swipe progress: ${progress}%`)}>
           <div>See on Page</div>
       </SwipeableListItem>

</SwipeableList>`

It's possible?
How is it done?

**Comment by marekrozmus** [comment]
Hi, you can make action to look like this:
```javascript
action: ( ) => window.open('https://www.google.com', '_blank')
```
